### PR TITLE
Added my profile (akshithbellare.json)

### PR DESCRIPTION
**Pull request by AkshithBellare**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

All tests passed
<p align="center">All existing tests passed</p>

My Profile page
<p align="center">My profile page</p>

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1681"></a>

### CODE REVIEW

**Review by SaptarshiSarkar12 — APPROVED** [review]
LGTM 👍. The changes can be merged.

**Comment by SaptarshiSarkar12** [comment]
@kumarsonsoff3 Please add `chore: profile addition` label.

**Review by Kunal0007 — APPROVED** [review]

File: public/data/akshithbellare.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "url": "https://twitter.com/AKSHITHBELLARE",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn 🤝",
+      "url": "https://www.linkedin.com/in/AKSHITHBELLARE",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Go to my Youtube! 🎥",
```
Comment (Panquesito7) [inline public/data/akshithbellare.json:28]: I recommend capitalizing the `t` here. What do you think?
```suggestion
      "name": "Go to my YouTube! 🎥",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by maharshi-sinha** [comment]
LGTM 🚀

**Comment by SaptarshiSarkar12** [comment]
@Panquesito7 This PR can now be merged.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**